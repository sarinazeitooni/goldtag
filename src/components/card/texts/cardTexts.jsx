const cardTexts ={
    purchased : 'فروخته شد',
    currency : 'تومان'
};
export default cardTexts;