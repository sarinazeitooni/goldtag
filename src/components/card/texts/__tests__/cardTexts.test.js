import React from 'react';
import { shallow } from 'enzyme';
import cardTexts from "../cardTexts";
describe("cardTexts text" , ()=>{
    it("it should render a div",()=>{
        const container = shallow(<cardTexts/>)
        expect(container).toMatchSnapshot();
    });
});