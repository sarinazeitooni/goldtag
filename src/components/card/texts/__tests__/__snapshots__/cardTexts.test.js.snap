// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`cardTexts text it should render a div 1`] = `ShallowWrapper {}`;
